docs/android/widgets/simpleWidget.md

```diff
@@ -8,8 +8,11 @@ permalink: /docs/android/widgets/simple/
 ---
 
 # Simple Widget
-
 {: .no_toc }
+<small>cartrawler.core.ui.views.partner.CTSimpleWidget</small>
+
```

Comment:
Maybe this one could be highlighted a bit more cause it is the main thing we need when adding the widget
"cartrawler.core.ui.views.partner.CTSimpleWidget" just make the font darker and bigger would help

Reply:
Make it bold and or remove the <small> tags?

Reply:
Yeah I think that should be enough, cause I nearly missed it when looking at it cause it is really light grey